src/views/components/common/trace-statistics-table/trace-item.vue

```diff
@@ -0,0 +1,84 @@
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+<template>
+  <div>
+    <div :class="['trace-item']" ref="traceItem">
+      <div :class="['method']" :style="{ 'text-indent': 10 + 'px', width: `${method}px` }">
+        <span v-tooltip:bottom="{ content: data.endpointName, popperCls: ['trace-table-tooltip'] }">
+          {{ data.endpointName }}
+        </span>
+      </div>
+
+      <div class="max-time">
+        {{ data.maxTime }}
+      </div>
+      <div class="avg-time">
+        {{ parseInt(data.avgTime) }}
+      </div>
+      <div class="min-time">
+        {{ data.minTime }}
+      </div>
+      <div class="count">
+        {{ data.count }}
+      </div>
+      <div class="sum-time">
+        {{ data.sumTime }}
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="js">
+  export default {
```

Comment:
As above. Please reuse existing components.

Reply:
> As above. Please reuse existing components.

We have merge code in trace.scss,trace-constant.ts.
But in trace-container.vue, it shows different table columns, and use v-if is not a good idead,we decide to rname and see new commit